source/node/introduction.rst

```diff
@@ -41,18 +41,21 @@ ethereum keystore.
 Session
 ^^^^^^^
 
-Provide an environment for the user access his ID provider, key manager and
-manage all the channels, throughout their lifecycle. In the context of a
-session, he/she can open a state channel, do some transactions through channels
-and settle the channel. Each session runs its own instance of state channel
-network client, ID provider, key manager; and hence provides a complete
-isolated environment within the node. It is this feature that enables multiple
-users to use the same node by having dedicated sessions for each of them.
+Provide an environment for the user to access his ID provider, key manager and
+all the channels opened within this session. In the context of a session,
+he/she can open a state channel, do some transactions on the channels and
+settle it. Each session runs its own instance of state channel network client,
+ID provider, key manager; and hence provides a complete isolated environment
```

Comment:
```suggestion
ID provider, key manager and hence provides a complete isolated environment
```

Reply:
I used `;` to separate the clauses (as described in [this example](https://www.merriam-webster.com/words-at-play/a-guide-to-using-semicolons)).

Reply:
okay